projects/core/src/lib/components/home-page/home-page.component.scss

```diff
@@ -0,0 +1,56 @@
+%text-center {
+	text-align: center;
+}
```

Comment:
I don't see the practical use of an additional inheritance abstraction for text-align: center; alone. This approach only increases the amount of code

Reply:
resolved